test/integration/manager-discovery.yaml.tmpl

```diff
@@ -27,9 +27,14 @@ spec:
       - name: manager
         image: {{.hub}}/runtime:{{.tag}}
         imagePullPolicy: Always
-        args: ["discovery", "-v", "2", "-n", "{{.namespace}}"]
+        args: ["discovery", "-v", "2", "-n", "$(POD_NAMESPACE)"]
         ports:
         - containerPort: 8080
+        env:
```

Comment:
Ok, will remove this. For clarification, -n flag is used to filter the k8s namespaces the manager watches for events as pod, svc creation/deletion etc. This should be the namespace  (or a list of namespaces) where the apps are deployed, and not the namespace where istio infra is deployed.